src/init/starship.zsh

```diff
@@ -63,12 +69,12 @@ starship_zle-keymap-select() {
 local existing_keymap_select_fn=$widgets[zle-keymap-select];
 # zle-keymap-select is a special widget so it'll be "user:fnName" or nothing. Let's get fnName only.
 existing_keymap_select_fn=${existing_keymap_select_fn//user:};
-if [[ -z ${existing_keymap_select_fn} ]]; then
+if [[ -z $existing_keymap_select_fn ]]; then
```

Comment:
Is it really needed to remove the braces (here and elsewhere)? At least I consider using braces for shell vars (`${VAR}` instead of ~`$VAR`~) a best practise.

Reply:
I don't have a strong opinion either way. Most variables in that file don't use `$`?

Reply:
@jankatins I removed those for sake of consistency. You can either always use curlies or only where needed and here was the latter case so I simply followed the suit to make it easier for future maintainers.

Regarding whether the extensive use of curlies is a best practice well, that's certainly a matter of personal preference but I'd argue it is not. Personally, I'm using them only for interpolations and stuff like length calculating, indexing, expansions otherwise I avoid them to minimize the clutter. I pretty much do what Mr. Shellharden _preaches_ https://github.com/anordal/shellharden/blob/master/how_to_do_things_safely_in_bash.md#should-i-use-curly-braces